scripts/build/rollup.config.js

```diff
@@ -0,0 +1,108 @@
+const resolve = require('rollup-plugin-node-resolve')
+const { uglify } = require('rollup-plugin-uglify')
+const babel = require('rollup-plugin-babel')
+const alias = require('rollup-plugin-alias')
+const cjs = require('rollup-plugin-commonjs')
+const replace = require('rollup-plugin-replace')
+const path = require('path')
+const lernaAliases = require('lerna-alias').rollup
+
+// this makes sure nested imports of external packages are external
+const makeExternalPredicate = externalArr => {
+  if (externalArr.length === 0) {
+    return () => false
+  }
+  const pattern = new RegExp(`^(${externalArr.join('|')})($|/)`)
+  return id => pattern.test(id)
+}
+
+function getChildPeerDeps(finalPeerDeps, depKeys) {
+  depKeys.forEach(key => {
+    const pkgJson = require(key + '/package.json')
+    if (pkgJson.peerDependencies) {
+      finalPeerDeps.push(...Object.keys(pkgJson.peerDependencies))
+      getChildPeerDeps(finalPeerDeps, Object.keys(pkgJson.peerDependencies))
+    }
+    // if (pkgJson.dependencies) {
+    //   getChildPeerDeps(finalPeerDeps, Object.keys(pkgJson.dependencies))
+    // }
+  })
+}
+
+module.exports = (data, isUMD = false, isBrowser = false) => {
+  const { pkg } = data
+  let external = []
+  if (pkg.peerDependencies) {
+    external.push(...Object.keys(pkg.peerDependencies))
+  }
+  if (pkg.dependencies && !isUMD) {
+    external.push(...Object.keys(pkg.dependencies))
+  }
+  getChildPeerDeps(external, external)
+  external.push('fs', 'path')
+  if (data.name === 'react-emotion') {
+    external = external.filter(name => name !== 'emotion')
+  }
+
+  const config = {
+    input: path.resolve(data.path, 'src', 'index.js'),
+    external: makeExternalPredicate(external),
+    plugins: [
+      babel({
+        presets: [
+          [
+            '@babel/env',
+            {
+              loose: true,
+              modules: false,
+              exclude: ['transform-typeof-symbol']
+            }
+          ],
+          '@babel/react',
+          '@babel/flow'
+        ],
+        plugins: [
+          '@babel/plugin-transform-flow-strip-types',
+          require('./add-basic-constructor-to-react-component'),
+          'codegen',
+          ['@babel/proposal-class-properties', { loose: true }],
+          require('./fix-dce-for-classes-with-statics'),
+          isBrowser && require('./inline-isBrowser'),
+          isBrowser &&
```

Comment:
isnt this duplicating what `inline-isBrowser.js` do (except filtering imported specifiers)? why there are 2 similar plugins?

Reply:
`inline-isBrowser.js` is replacing the usages of `@emotion/utils` in packages whereas this plugin is changing the declarations inside of `@emotion/utils`

Reply:
could we maybe add a filename guard to this second plugin so it wont run accidentally on other packages? I think there are some nice new options in babel@7 - like `overrides` etc, maybe they could be leveraged to add this plugin only for certain filepaths/directories, this would make the intent more explicit